### ISSUE TITLE
[3.5.0] [Vite] cannot clone body after it is used

### ISSUE THREAD

**Issue by halljus** [Code-Forge-Net/Remix-Dev-Tools#76]
Took 3.5.0 out for a test drive with Vite and it crashed when I submitted a form.

Stacktrace:

```
Error: cannot clone body after it is used
    at clone (C:\Users\user\dev\project\node_modules\@remix-run\web-fetch\dist\lib.node.cjs:588:9)
    at new Request (C:\Users\user\dev\project\node_modules\@remix-run\web-fetch\dist\lib.node.cjs:1456:6)
    at Request.clone (C:\Users\user\dev\project\node_modules\@remix-run\web-fetch\dist\lib.node.cjs:1604:10)
    at extractDataFromResponseOrRequest (file:///C:/Users/user/dev/project/node_modules/remix-development-tools/dist/server.js:702:32)
    at storeAndEmitActionOrLoaderInfo (file:///C:/Users/user/dev/project/node_modules/remix-development-tools/dist/server.js:732:26)
    at file:///C:/Users/user/dev/project/node_modules/remix-development-tools/dist/server.js:774:7
    at unAwaited (file:///C:/Users/user/dev/project/node_modules/remix-development-tools/dist/server.js:667:3)
    at file:///C:/Users/user/dev/project/node_modules/remix-development-tools/dist/server.js:772:5
```

**Comment by dong-qian** [comment]
The error occurred in the `﻿extractDataFromResponseOrRequest` function.

The problem lies in using ﻿Response.clone() which can only be used once. It is being used on the loader, and then the subsequent form post triggers an action which results in an error being thrown.

Ref: 
https://developer.mozilla.org/en-US/docs/Web/API/Response/clone


I tested the below fix, but no 100% sure it's the best way.

Replace `const extractable = response.clone();` with `const extractable = new Response(response.body, response);`

**Comment by AlemTuzlak** [comment]
@dong-qian Thank you so much! Merged and released in 3.5.1
